charts/kafka-connect/templates/kafka-connect.yaml

```diff
@@ -1,18 +1,14 @@
 apiVersion: kafka.strimzi.io/v1beta2
 kind: KafkaConnect
 metadata:
-  name: kafka
+  name: {{ .Values.metadataName }}
```

Comment:
`metadataName` is the name of the main value the variable sets. Open for suggestions for a better name,